tests/garage/envs/test_point_env.py

```diff
@@ -0,0 +1,13 @@
+import pickle
+import unittest
+
+from garage.envs.point_env import PointEnv
+from tests.helpers import step_env
+
+
+class TestPointEnv(unittest.TestCase):
+    def test_pickleable(self):
+        env = PointEnv()
+        round_trip = pickle.loads(pickle.dumps(env))
+        assert round_trip
```

Comment:
Are you just asserting round_trip exists because there are no parameters in env that you can compare equality to?

Reply:
This question also extends to all of your other tests that only have `assert round_trip` and no other assertions.

Reply:
Trivially we want to make sure any object was unpickled at all.